src/pages/Index.vue

```diff
@@ -1,4 +1,4 @@
-<template>
+<template onload="addDateListener()">
```

Comment:
Is this the best approach ?

Reply:
Try reading [this](https://stackoverflow.com/questions/50720802/how-to-set-the-default-date-of-the-day-in-vue-js)

Reply:
@jseanpatel I went ahead and took care of this.